types/jasmine/tslint.json

```diff
@@ -7,6 +7,7 @@
         "no-single-declare-module": false,
         "no-unnecessary-generics": false,
         "no-var-keyword": false,
+        "npm-naming": false,
```

Comment:
What does this do?

Reply:
If you run `npm lint` without this, you will get an error about how jasmine isn't a found npm package. I forget the exact error text.

Reply:
```
$ npm run lint angular-material

> definitely-typed@0.0.3 lint /DefinitelyTyped
> dtslint types "angular-material"

Error: /DefinitelyTyped/types/angular-material/index.d.ts:1:1
ERROR: 1:1  npm-naming  d.ts file must have a matching npm package.
To resolve this error, either:
1. Change the name to match an npm package.
2. Add a Definitely Typed header with the first line


    // Type definitions for non-npm package angular-material-browser

Add -browser to the end of your name to make sure it doesn't conflict with existing npm packages.

- OR -

3. Explicitly provide dts-critic with a source file. This is not allowed for submission to Definitely Typed.
 See: https://github.com/Microsoft/dtslint/blob/master/docs/npm-naming.md

    at /DefinitelyTyped/node_modules/dtslint/bin/index.js:190:19
    at Generator.next (<anonymous>)
    at fulfilled (/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! definitely-typed@0.0.3 lint: `dtslint types "angular-material"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the definitely-typed@0.0.3 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /.npm/_logs/2019-09-26T17_08_18_356Z-debug.log
```
That's the error you get if you don't disable `npm-naming`, I ran it on a different package that I also use.

I found that there is in fact an NPM package https://www.npmjs.com/package/jasmine, but why the linter still fails, who knows.